tests/unit/test_windows_compatibility.py

```
@@ -0,0 +1,262 @@
+        dangerous_commands = [
+            "format C:",
+            "del /S C:\\Windows",
+            "rd /S C:\\Users",
+            "reg delete HKLM\\Software",
+            "taskkill /F /T",
+            "Remove-Item -Recurse C:\\Windows",
+            "Stop-Computer -Force",
+        ]
+
+        for cmd in dangerous_commands:
+            is_safe, reason, cleaned = sanitizer.sanitize_command(cmd)
```

Pylint (3.3.7)

[warning] Unused variable 'cleaned'

(W0612)